src/openms/include/OpenMS/ANALYSIS/TARGETED/TargetedExperimentHelper.h

```diff
@@ -222,9 +268,10 @@ namespace OpenMS
       };
 
       Peptide() :
-        CVTermList()
+        CVTermList(),
+        charge_(0),
+        charge_set_(false)
       {
-        charge_ = -1;
```

Comment:
this wont work any more as metabolites can have negative charge. Also it would be pretty bad if the charge is _not_ set and we would just use `-1` or `0` and divide by that!